tailwind.config.ts

```diff
@@ -39,7 +40,7 @@ module.exports = {
           3: "var(--calcite-ui-foreground-3)"
         }
       },
-      text: {
+      color: {
```

Comment:
We might be able to remove or refactor this at some point. Tailwind colors usually just go like `text-<color>` instead of `text-color-<color>`. This allows people to use colors as variants